components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/OAuthAdminServiceImplTest.java

```diff
@@ -112,6 +119,14 @@ public class OAuthAdminServiceImplTest extends PowerMockIdentityBaseTest {
     private AxisConfiguration axisConfiguration;
     @Mock
     private TenantManager tenantManager;
+    @Mock
+    Tenant mockTenant;
+    @Mock
+    UserRealm mockUserRealmFromRealmService;
```

Comment:
do we really need this? can't we use https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1881/files#diff-44570cdf90cc2f9386a1cfbc38b598bdd93a2d2a04d7bdfda4b9c1ea0bffeb97R111?

Reply:
resolved